packages/@ember/-internals/glimmer/tests/integration/components/runtime-template-compiler-implicit-test.ts

```diff
@@ -7,6 +7,29 @@ import { fn } from '@ember/helper';
 moduleFor(
   'Strict Mode - Runtime Template Compiler (implicit)',
   class extends RenderingTestCase {
+    async '@test can immediately render a runtime-compiled template'() {
+      class State {
+        get component() {
+          return template(`hello there`);
+        }
+      }
+
+      let state = new State();
+
+      hide(state);
```

Comment:
@ef4 Now we know `hide` isn't just for eslint

Reply:
I think somewhere it was document (on hide maybe?) that this is for tricking JIT stuff